definitions/features/upstream_repositories.rb

```diff
@@ -14,6 +14,36 @@ def repoids_and_urls
     repoids_and_urls
   end
 
+  VERSION_MAPPING = {
+    '3.2' => '4.4',
+    '3.3' => '4.5'
+  }.freeze
+
+  def server_url
+    if el?
+      return 'https://yum.theforeman.org/'
+    end
+
+    'https://deb.theforeman.org/'
+  end
+
+  def foreman_release_pkg_url(version)
+    if el?
+      return "#{server_url}/releases/#{version}/#{el_short_name}/x86_64/foreman-release.rpm"
+    end
+
+    "#{server_url}/pool/#{os_version_codename}/#{version}/f/foreman-release/foreman-release.deb"
```

Comment:
Do we have some symlink for latest deb package of foreman-release? if not then is that what needs to be changed in packaging?